client.js

```diff
@@ -303,6 +308,53 @@ function getHand(playerTable, player) {
   console.log('ERR: requested player wasn\'t in player table')
 }
 
+// Order hands in updated game to match the order before the update
+function orderHand(oldGame, newGame) {
+  for (i = 0; i < oldGame.players.length; i++) {
+    var oldHand = oldGame.players[i][1]
+    var newHand = newGame.players[i][1]
+
+    var orderedHand = []
+
+    // If a letter of the new hand was also in the pre-update hand, then put
+    // it in the same location. Otherwise, add it to the end.
+    for (const letter of oldHand) {
+      idx = newHand.indexOf(letter)
```

Comment:
how does this work when there's two of the same letter in the hand?

Reply:
like if old hand is ATOOTVZ, wouldn't both Ts try to go into the position of the first T that indexOf comes across in the oldHand (at position 1)? So as you added each item of the oldHand to the newHand you'd get A -> AT -> ATO -> ATOO -> ATTOO -> ATTOOV -> ATTOOVZ

Reply:
I think it still works because it's not inserting into `orderedHand` (the final, sorted, new hand) by index. So once you have ATOO it can't add the second T anywhere but at the end.

Reply:
ok i don't get it but it seems to work :D